src/UmbracoContentApi/UmbracoContentApi.Web/Controllers/PreviewApiController.cs

```diff
@@ -36,7 +32,7 @@ public IHttpActionResult Get(Guid id)
         {
             var val = _publishedSnapshotService.EnterPreview(_userService.GetUserById(-1), _contentService.GetById(id).Id);
```

Comment:
By the way, this is unused, I am not sure what your intention is so I have not removed it.

Reply:
The intent for this is to create preview functionality so you can preview content when using Umbraco as Headless api. It was more of a test thing to see if I can get saved, but not published, content instead of the published version. If you have any idea how I might be able to do that it would be greatly appreciated :-)

Reply:
I see a few `_contentService` references. I would only warn you to be very very careful with those. I don't really know how to deal with Previews I'm afraid. 😅

Reply:
Ah yeah I will be careful with them. Might need to make the documentation more clear that the current package is only the Core implementation. The Web project is currently mostly meant for testing the Core :-p That said, I will definitely keep it in mind, as I do want to release the Web project as a separate package when I have the preview working :-)